common/collection/Bytes.java

```diff
@@ -169,10 +169,14 @@ public static double sortedBytesToDouble(byte[] bytes) {
     }
 
     public static String bytesToString(byte[] bytes, Charset encoding) {
-        final byte[] x = Arrays.copyOfRange(bytes, 1, 1 + bytes[0]);
+        final byte[] x = Arrays.copyOfRange(bytes, 1, 1 + unsignedByteToInt(bytes[0]));
         return new String(x, encoding);
     }
 
+    public static int unsignedByteToInt(byte x) {
+        return x & 0xff;
```

Comment:
Can we write `ByteBuffer.wrap(content).getInt()` instead to make it look less h4ck3r?

Reply:
I think this fine, and would be more performant, @lolski

Reply:
ok cool